videoclip.lua

```diff
@@ -469,4 +469,3 @@ end
 validate_config()
 encoder.init(config, main_menu.timings)
 mp.add_key_binding('c', 'videoclip-menu-open', main_menu.open)
-mp.msg.warn("Press 'c' to open the videoclip menu.")
```

Comment:
We need to keep this line to remind the user how to open the main menu.

Reply:
yes, i get that, but if you adjust your input.config, the text is no longer correct. Likewise, if you have already set an input to c, then it will not be correct either and not open the menu